Slambook/index.js

```diff
@@ -39,7 +39,7 @@ let nicknameValue= document.querySelector(`[name='nickname']`).value;
 let songForMeValue= document.querySelector(`[name='songForMe']`).value; 
 let shareOpinionValue= document.querySelector(`[name='shareOpinion']`).value;

```

Comment:
The 'allSlamArray' variable is not defined. Make sure to define it before using it.